ch01.adoc

```diff
@@ -77,7 +77,7 @@ When a data value provides information about conditions in a cell occupying a re
 CDL syntax:: The ascii format used to describe the contents of a netCDF file is called CDL (network Common Data form Language).
 This format represents arrays using the indexing conventions of the C programming language, i.e., index values start at 0, and in multidimensional arrays, when indexing over the elements of the array, it is the last declared dimension that is the fastest varying in terms of file storage order.
 The netCDF utilities ncdump and ncgen use this format (see
-link:$$http://www.unidata.ucar.edu/software/netcdf/docs/netcdf_utilities_guide.html#cdl_syntax$$[NUG section on CDL syntax]).
+link:$$https://www.unidata.ucar.edu/software/netcdf/docs/netcdf_utilities_guide.html#cdl_syntax$$[NUG section on CDL syntax]).
```

Comment:
Could this link do the job https://docs.unidata.ucar.edu/nug/current/_c_d_l.html ?

Reply:
Hi @larsbarring, thanks for all of your comments here.

Good find. That link does seem comparable to me. Is it best, admin logistics wise, for me to update the link here in this PR, and similarly for the other one below, or should we request these link changes via another Issue/PR set?

Reply:
Let's keep it simple and just update here because these changes are just housekeeping matters to have a nice and tidy document, and repo. Then add a comment to the history (see below).

Reply:
Good idea, thanks again.

Link updated to the above in [6cb0d99](https://github.com/cf-convention/cf-conventions/pull/476/commits/6cb0d99cbb119ef31b5da779bab321f8f7f90a83), so marking this as resolved.